src/cif.cc

```diff
@@ -6,7 +10,14 @@
 #include <gemmi/cif.hpp>
 #include <gemmi/mmcif.hpp>
 
+#define GEMMI_WRITE_IMPLEMENTATION
+#include <gemmi/to_cif.hpp>
+#undef GEMMI_WRITE_IMPLEMENTATION
```

Comment:
This could probably be causing the problems in CI

Reply:
I get the same problem on my computer, so they're probably not the cause. But that makes it easier to find the source.